### DISCUSSION TITLE
looking for collaborators

### DISCUSSION THREAD

**Discussion by MerylStreep1** [community/community#141000] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am looking for collaborators for this project if you want to ask me questions you can write to me privately     https://github.com/MerylStreep1/Bypass-Instagram-Message-Requests-and-Send-Direct-Messages

**Comment by LiteBrite82** [comment]
Hi @MerylStreep1 , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
